Makefile

```diff
@@ -74,10 +72,10 @@ clippy-nmea:
 	cargo clippy -p micro-rdk-nmea --no-deps --features native -- -Dwarnings
 
 clippy-native:
-	cargo clippy -p micro-rdk --no-deps --features native,ota  -- -Dwarnings
+	cargo clippy -p micro-rdk --no-deps --features native,ota --release  -- -Dwarnings
```

Comment:
Just FYI but we have some overlapping/conflicting changes between this and #430.